readme.md

```diff
@@ -84,52 +84,49 @@ your_method(){
 }
 ```
 
-# API Referenced
-
-> ## **setLanguage** ( lang ) : void
+## API References
 
-Set language default. if lang is null, string will read from default.json file. else, it will read from [lang].str
+#### .setLanguage(lang): void
 
-### **params**
+Set default language. If lang is null, string will be read from **default.json** file. Otherwise, it will be read from **<lang>.str**
 
-- **lang** ( string ) : language code, it should be 2 digits. Example : en -> english, vi -> vietnamese. This code need coincides with the declared file in the strings directory
+**Parameters**
 
-> ## **set** ( key, val, lang? ) : void
+- `lang` _(string)_: language code, it should be exactly two letters. Example: en -> english, vi -> vietnamese. You should ensure that the used language are well reclared in **./strings** directory. For more conventional, you should use country codes defined by *ISO 3166-1 alpha-2*.
 
-set string like temporary in runtime
+#### .set(key, val, lang?): void
 
-### **params**
+Temporarily set string in runtime. The original file is left intact.
 
-- **key** ( string ) : string key, declared in .``str`` file
-- **val** ( object ) : parameters passed in string value
-- **lang** ( string ) : language code, declared in ``strings`` file
+**Parameters**
 
-> ## **get** ( key, args? ) : string
+- `key` _(string)_: string key, declared in `.str` file.
+- `val` _(object)_: parameters passed in string value.
+- `lang` _(string)_: language code, declared in string files.
 
+#### .get (key, args?): string
 
 Fill source string by target string.
 
-Stringer support internal reference represent with syntax : <#[abort-key-name]()>.
+Stringer support internal reference represent with syntax: `#[abort-key-name]()`.
```

Comment:
i'm used syntax of it is <#field_name> to reference to the field that declared abort in .str file